align_system/algorithms/oracle_adm.py

```diff
@@ -17,30 +15,15 @@
 
 
 class OracleADM(ActionBasedADM):
-    def __init__(
-        self, probabilistic: bool=False, upweight_missing_kdmas: bool=False,
-        filter_unlabeled_actions: bool=False, misaligned: bool=False, **kwargs
-    ):
-        self.probabilistic = probabilistic
-        self.upweight_missing_kdmas = upweight_missing_kdmas
-        self.filter_unlabeled_actions = filter_unlabeled_actions
+    def __init__(self, misaligned: bool=False, **kwargs):
```

Comment:
Missed this in my "official" review, but several of these init arguments (probabilistic etc.) are ignored now during initialization (and are now expected to be passed in for the inference_kwargs is that right?), do any of the config files need to be updated to reflect this change?  Also if we're not actually using the `**kwargs` in the init method I think we should remove it since folks will at least get an error if they pass in extraneous arguments

Reply:
I just updated all the configs of the adms I touched. 

For the oracle in particular, we lost the `upweight_missing_kdmas`  (no longer an option) and `filter_unlabeled_actions` (always on) functionality when we switched to the separate alignment utils. So this was fully removing those since they aren't true options anymore

When I was rebasing this I realized I had switched probabilistic to an inference kwarg, when you were just talking about switching things over to instance args instead, so I can make that change if you prefer

Reply:
Gotcha.  Yeah slight preference for moving it (`probabilistic`) back to the init arguments.  Thanks

Reply:
Done!